lib/_stream_readable.js

```diff
@@ -1043,7 +1045,7 @@ function endReadable(stream) {
   // If we get here before consuming all the bytes, then that is a
   // bug in node.  Should never happen.
   if (state.length > 0)
-    throw new Error('"endReadable()" called on non-empty stream');
+    throw new errors.Error('ERR_STREAM_READABLE_CALLED_ON_NONEMPTY');
```

Comment:
This should be `ERR_STREAM_ENDREADABLE_CALLED_ON_NONEMPTY`.

Reply:
I've updated this line